templates/slack/answer-issued.ejs

```diff
@@ -1,4 +1,4 @@
-*`<%- notification.space.ens %>`: 💰 New Reality Answer!*
+*<%- notification.space.ens %>: 💰 New Reality Answer!*
```

Comment:
I would love to merge this ASAP so it is ready in the next deployment. Do you mind issuing a different PR for that? (I can replicate the changes if that makes it easier)